packages/caravan-health/src/privacy.test.ts

```diff
@@ -282,40 +315,65 @@ describe("Privacy metric scoring", () => {
 
   describe("Transaction Topology Score", () => {
     it("Calculates the transaction topology score based on the spend type", () => {
+      // not a self payment
       const score: number = privacyMetric.getTopologyScore(transactions[0]);
-      expect(score).toBe(0.75);
+      expect(score).toBe(0.5);
```

Comment:
see, now we have a different outcome. maybe having both is fine, but we shouldn't be losing a test case imo.